src/main/java/org/killbill/billing/plugin/adyen/client/model/PurchaseResult.java

```diff
@@ -118,21 +121,26 @@ public String getResultCode() {
         return errorCodes;
     }
 
+    public Optional<AdyenCallErrorStatus> getAdyenCallErrorStatus() {
```

Comment:
I brought it up because I would have expected the initial payment and follow-up transaction flows to be symmetric, but it doesn't look like it's the case?